scripts/video_generator.py

```diff
@@ -3,10 +3,12 @@
 from video import create_segment, concatenate_segments
 from narration import create_narration
 from watermark import add_watermark
+from config import Config
+cfg = Config()
 
 def generate():
     # Read the YAML file
-    with open("input/video_script.yml", "r") as file:
+    with open(cfg.script_path, "r") as file:
```

Comment:
Will this change not always use the`"input/sample_script.yml"` mentioned above? Since is hard coded into the config